code/__defines/gamemode.dm

```diff
@@ -1,3 +1,5 @@
+
+#define GAME_STATE_STARTUP		0
```

Comment:
what is this new game state? when is it active?

Reply:
https://github.com/discordia-space/CEV-Eris/pull/1874/files#diff-217023bb5cfd5c76d3b8403cfaa58f7aR64 initial state for stuff, that was outside of current_state in old gameticker.